homeassistant/util/yaml.py

```diff
@@ -6,6 +6,7 @@
 import glob
 import yaml
 
+import homeassistant.util.secrets as secrets
```

Comment:
Importing keyring in yaml.py is fine as long as you keep same behavior so it is not used if not available.

Let's try to solve one thing at a time. If we want to add a get_secret for components, we could add that in the future. It is not straight forward tho, what if you want to load a platform twice, each with another account. So let's keep that out of this PR too.